### DISCUSSION TITLE
How to pass more configs to the xmake project init?

### DISCUSSION THREAD

**Discussion by fripSide** [xmake-io/xmake#2129] [Q&A]
I need to pass other paths or configs to xmake.lua just like the --sdk or --ndk. Such that, I can set user-specific configs without using ENV. 
```
xmake f -my_config=xxxxx
```
Might I add an option for it?

**Comment by waruqi** [comment]
For cross-compiling the toolchain, you just need to use the existing `--sdk=` option, otherwise please add a custom option.
